features/tab-access-tabs.feature

```diff
@@ -0,0 +1,79 @@
+Feature: Get or set TabStops
+  In order to customize tab stops
+  As a python-docx developer
+  I need a TabStops object with read/write tab_stop properties
+
+
+  @wip
+  Scenario Outline: Get tab_stop objects
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops contain tab_stop objects
+      
+    Examples:  values for tab_stop elements 
+      | value |
+      | some  |
+      
+
+  @wip
+  Scenario Outline: Count tab stop objects
+    Given a paragraph having <value> tab_stop elements
+      Then paragraph_format.tab_stops has <lenval> length
+      
+    Examples:  length values for tab_stop elements
+      | value | lenval |
+      | no    | 0      |
+      | some  | 3      |
+      
```

Comment:
* I like the "Count ..." phrase, I think I'll adopt that one going forward for len() :)

* the given should be a tab_stops object (direct parent), not a paragraph (great grandparent). Let's use the phrase "Given a tab_stops ..." for this and the others too.

* I think "count" works better than "value" in this case. The values should be numeric, not text, and definitely not ambiguous (as is "some"). I'd say 0, 1, and 3, since those correspond to the classic cases zero, one, and more than one. You should be able to just have a count column in the examples then since no mapping is necessary.

* the then clause should be: "Then len(tab_stops) is <count>"